packages/flat-config-disable-type-checked/eslint.config.js

```diff
@@ -16,7 +16,7 @@ export default tseslint.config(
     },
   },
   {
+    extends: [tseslint.configs.disableTypeChecked],
     files: ["*.js"],
-    ...tseslint.configs.disableTypeChecked,
```

Comment:
`extends` actually always worked this way, I'd just not gotten in the habit of using it nicely.